front/pages/w/[wId]/builder/data-sources/static.tsx

```diff
@@ -223,9 +227,17 @@ function getTableColumns() {
       meta: {
         width: "6rem",
       },
+      sortingFn: (rowA: Row<RowData>, rowB: Row<RowData>) => {
+        return (
+          (rowA.original.usage.count ?? 0) - (rowB.original.usage.count ?? 0)
+        );
+      },
       cell: (info: Info) => (
-        <DataTable.CellContent icon={RobotIcon}>
-          {info.row.original.usage}
+        <DataTable.CellContent
+          icon={RobotIcon}
+          title={`Used by ${info.row.original.usage.agentNames.join(", ")}`}
```

Comment:
noob question: can't recall if `DataTable.CellContent` does this ellipsis itself?

Reply:
It's a "title" (straight from html, so there is no truncating).
I couldn't use Tooltip as there are some layouting issue and it's clipped (and I don't want to spend 300h fixing css).

<img width="1013" alt="image" src="https://github.com/user-attachments/assets/fe60163d-f71c-44cc-b9f7-36d1393dcca1">